frontend/src/customPropTypes/documents.js

```diff
@@ -0,0 +1,24 @@
+import { PropTypes, shape, arrayOf } from "prop-types";
+
+// This file is anticipated to have multiple exports
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
This can be deleted now as the file contains multiple exports and ESLint is no longer mad.